### PR TITLE
Provide a method for obtaining a link to justification description

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

```python
>>> from thoth.common import get_justification_link as jl
>>> jl("oom")
'https://thoth-station.ninja/justifications/oom.html'
```

### CODE REVIEW

File: thoth/common/helpers.py [hunk 1]
```diff
@@ -169,3 +169,8 @@ def __get__(self, obj: object, _: Any = None) -> Any:
         value = self._calculate(obj)
         setattr(obj, self._calculate.__name__, value)
         return value
+
+
+def get_justification_link(identifier: str) -> str:
+    """Construct a link to a detailed justification document."""
+    return f"https://thoth-station.ninja/justifications/{identifier}.html"
```
Comment (pacospace) [inline thoth/common/helpers.py:176]: don't we want this to be an env variable as maybe we are going to change something in the future?

**Review by pacospace — COMMENTED** [review]
One small comment, otherwise very nice!!

File: thoth/common/helpers.py (hunk 1 above)
Comment (fridex) [reply thoth/common/helpers.py]: Good idea, done in f4287ea

**Review by pacospace — APPROVED** [review]
:100 Thanks!

**Pull request merged**